bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -314,6 +314,7 @@
       <a rel="external" href="https://blog.mozilla.org/press/kits/">{{ _('Press Kit') }}</a>
       <a rel="external" href="https://blog.mozilla.org/press/">{{ _('Mozilla and Firefox News') }}</a>
       <a href="{{ url('firefox.organizations.organizations') }}">{{ _('Firefox Extended Support Release') }}</a>
+      <a href="{{ url('firefox.releases.index') }}">{{ _('All Firefox Releases') }}</a>
```

Comment:
Since suggesting this copy I remembered the link lists desktop releases only, so to make it a little clearer `All Firefox for desktop releases` might be a little clearer.